Content.Client/Ninja/NinjaSystem.cs

```diff
@@ -0,0 +1,14 @@
+using Content.Client.Interactable.Components;
+using Content.Shared.Ninja.Components;
+using Content.Shared.Ninja.Systems;
+using Content.Shared.Stealth.Components;
+
+namespace Content.Client.Ninja;
+
+/// <summary>
+/// Currently does nothing special clientside.
+/// All functionality is in shared and server.
```

Comment:
Is there anything in the server code that can't be in the Shared Code? If so you can nuke this and server while having everything in Shared.

Reply:
afaik actions have to be serverside only
also i have changed the systems thing locally each thing has its own so thatll be fun